ci/before_deploy.sh

```diff
@@ -0,0 +1,33 @@
+# This script takes care of building your crate and packaging it for release
+
+set -ex
+
+main() {
+    local src=$(pwd) \
+          stage=
+
+    case $TRAVIS_OS_NAME in
+        linux)
+            stage=$(mktemp -d)
+            ;;
+        osx)
+            stage=$(mktemp -d -t tmp)
+            ;;
+    esac
+
+    test -f Cargo.lock || cargo generate-lockfile
+
+    # TODO: this should build the ci and so files
+    #cross rustc --bin hello --target $TARGET --release -- -C lto
+
+    # TODO: this should copy the ci and so files
+    cp target/$TARGET/release/hello $stage/
```

Comment:
I don't think we have a `release/hello` file, so this should be update or commented out.

Reply:
oh right, that was supposed to be for the ci file

Reply:
commenting out for now